composer.json

```
@@ -13,26 +13,38 @@
     "prefer-stable": true,
     "require": {
         "php": ">=8.1",
-        "doctrine/doctrine-bundle": "^2.10.0",
         "makinacorpus/query-builder": "^1.6.1",
+        "psr/log": "^3.0",
         "symfony/config": "^6.0|^7.0",
-        "symfony/console": "^6.0|^7.0",
-        "symfony/dependency-injection": "^6.0|^7.0",
         "symfony/filesystem": "^6.0|^7.0",
         "symfony/finder": "^6.0|^7.0",
         "symfony/options-resolver": "^6.0|^7.0",
-        "symfony/password-hasher": "^6.0|^7.0",
         "symfony/process": "^6.0|^7.0",
         "symfony/yaml": "^6.0|^7.0"
     },
     "require-dev": {
+        "doctrine/doctrine-bundle": "^2.10.0",
         "doctrine/orm": "^2.15|^3.0",
         "friendsofphp/php-cs-fixer": "^3.34",
         "phpstan/phpstan": "^1.10",
         "phpunit/phpunit": "^10.4",
+        "symfony/console": "^6.0|^7.0",
+        "symfony/dependency-injection": "^6.0|^7.0",
         "symfony/framework-bundle": "^6.0|^7.0",
+        "symfony/password-hasher": "^6.0|^7.0",
         "symfony/validator": "^6.3|^7.0"
     },
+    "suggest": {
+        "composer/pcre": "For PHAR compilation",
+        "doctrine/doctrine-bundle": "For autoconfiguration in Symfony project context",
+        "seld/phar-utils": "For PHAR compilation",
+        "symfony/password-hasher": "In order to use the password hash anonymizer"
+    },
```

```suggestion
    "suggest": {
        "doctrine/doctrine-bundle": "For autoconfiguration in Symfony project context",
        "symfony/password-hasher": "In order to use the password hash anonymizer"
    },
```

I think phar compilation will be, at the end, made by a CI job and will only concern a few people. Thus I wouldn't suggest installing these dependencies.

---

Yes, that's a good point. I'm going to remove this ASAP.

---

Done.